### PR TITLE
Fixed ExpressionDoesNotDefineNullParameterWithoutNullOption test

### PR DESCRIPTION
**Pull request by Bykiev**
Fixed ExpressionDoesNotDefineNullParameterWithoutNullOption test

### CODE REVIEW

**Comment by Bykiev** [comment]
@sklose, please, review this text fix

File: test/NCalc.Tests/Fixtures.cs [hunk 1]
```diff
@@ -112,7 +112,7 @@ public void ExpressionDoesNotDefineNullParameterWithoutNullOption()
             var e = new Expression("'a string' == null");
 
             var ex = Assert.Throws<ArgumentException>(() => e.Evaluate());
-            Assert.Contains("Parameter name: null", ex.Message);
+            Assert.Contains("Parameter was not defined", ex.Message);
```
Comment (sklose) [inline test/NCalc.Tests/Fixtures.cs:115]: how come this test passed before and is still passing with the change? 🤔

Comment (Bykiev) [reply test/NCalc.Tests/Fixtures.cs:115]: Indeed, I don't know why does it passing on AppVeyor, but it fails in Visual Studio. Seems to be an issue on AppVeyor side...

Comment (sklose) [reply test/NCalc.Tests/Fixtures.cs:115]: Interesting  - sounds like something that needs to be fixed in CI

**Review by sklose — APPROVED** [review]

**Pull request merged**